src/psyclone/psyir/nodes/kernel_schedule.py

```diff
@@ -55,7 +55,6 @@ class KernelSchedule(Routine):
     def __init__(self, name, parent=None):
```

Comment:
The `__init__` and `create` methods of this class don't add anything and can be fully removed.

Reply:
Good spot, done. That does however mean that `KernelSchedule` adds nothing to `Routine` but it's probably useful to keep as a way of classifying the various API-specific Kernel classes?

Reply:
I agree. This node should be migrated to `domain/common` at some point.